src/Components/NoLokiSplash.tsx

```diff
@@ -10,13 +10,11 @@ export const NoLokiSplash = () => {
   return (
     <div className={styles.wrap}>
       <div className={styles.graphicContainer}>
-        <SVG
-          src={
-            theme.isDark
-              ? `/public/plugins/grafana-lokiexplore-app/img/grot_loki.svg`
-              : `/public/plugins/grafana-lokiexplore-app/img/grot_loki.svg`
-          }
-        />
+        {React.createElement(SVG as any, {
```

Comment:
I don't love the type assertion, plus let's get rid of this unnecessary ternary

Reply:
Strange I was getting a typescript error but after `yarn install` it has gone away and I'm able to keep using <SVG>